pkg/pillar/agentlog/eveversion.go

```diff
@@ -9,9 +9,15 @@ import (
 	"strings"
 )
 
-const versionFile = "/etc/eve-release"
+const (
+	versionFile   = "/etc/eve-release"
```

Comment:
We have to be careful since this doesn't work in the lisp (or any other non-pillar) container.

We should move the setting of /run/eve.id and /run/eve-release to something like the storage-init container, and then use /run/eve-release.

Reply:
Please fix this @gkodali-zededa

Reply:
The /run/eve-release is already there. We can separately potentially move it to storage-init

Reply:
@eriknordmark Bind mounting the /etc/release from host to lisp container is easier.

Reply:
Added copy of /etc/eve-release to /run/eve-release as part of dom0-ztools container.